backend/src/helpers/helpers.js

```diff
@@ -0,0 +1,41 @@
+// Priorities are set from the lowest to the highest.
+const statePriorities = [ 'away', 'viewing', 'commenting', 'editing', 'merging' ];
+
+module.exports = { getUserListFromSessions };
+
+function getUserListFromSessions( issueSessions ) {
```

Comment:
I like when a code has some space.

```
function getUserListFromSessions( issueSessions ) {
	const users = {};

	for ( const socketId in issueSessions ) {
		const issueSession = issueSessions[ socketId ];

 		if ( !users[ issueSession.id ] ) {
			users[ issueSession.id ] = [];
		}

		users[ issueSession.id ].push( issueSession );
	}

 	return Object.values( users )
		.map( userSessions => {
			return {
				...userSessions[ 0 ],
				state: userSessions
					.map( session => session.state )
					.reduce(
						( mostImportantState, nextState ) => chooseMoreImportantUserState( mostImportantState, nextState ),
						statePriorities[ 0 ]
					)
			};
		} )
		.sort( sortUsersByDate );
}
```

Reply:
Also, check other fragments of your code ;)

Reply:
Done, I've also checked other files.